spyder-remote-client/spyder_remote_client/spyder/widgets.py

```diff
@@ -226,4 +226,6 @@ def change_host(self):
             for key, value in properties.items():
                 if key.startswith("conda_env_") and key.endswith("_yes"):
                     env_name = value.split("/")[-1]
-                    self.env_comvo.addItem(env_name, value)
+                    #Fix for issue #27 for the spyder-remote-server issue 
+                    prefix = '/home/'+properties["guest_account"]+'/'+env_name
```

Comment:
@nerohmot It would be probably better to use the `conda_prefix` information on the `properties`and generate from there 🤔